recipe/meta.yaml

```diff
@@ -1,24 +1,17 @@
-{% set version="3.16.2" %}
-{% set year="2017" %}
-{% set version_split=version.split(".") %}
-{% set major=version_split[0] %}
-{% set minor=version_split[1]|int * 10 %}
-{% set bugfix=version_split[2]|int * 100 %}
-{% set version_coded=(major ~ (("%03d" % minor)|string) ~ (("%03d" % bugfix)|string)) %}
```

Comment:
Why was the version coding for the `url` dropped? :confused: This helps avoid issues like the one in PR ( https://github.com/conda-forge/sqlite-feedstock/pull/21 ).